NOTICE.txt

```diff
@@ -519,6 +519,9 @@ License notice for Apache Commons Codec
 Copyright 2002-2019 The Apache Software Foundation
 
 commons-codec (https://github.com/apache/commons-codec)
+commons-text (https://github.com/apache/commons-text)
```

Comment:
Should we update the title to -> 
`License notice for Apache Commons` instead of `License notice for Apache Commons Codec` ?

Reply:
Good catch, we should.